packages/lodestar/src/network/gossip/validator.ts

```diff
@@ -56,66 +57,68 @@ export class GossipMessageValidator implements IGossipMessageValidator {
     this.logger = logger;
   }
 
-  public isValidIncomingBlock = async (signedBlock: SignedBeaconBlock): Promise<boolean> => {
+  public isValidIncomingBlock = async (signedBlock: SignedBeaconBlock): Promise<ExtendedValidatorResult> => {
     const finalizedCheckpoint = await this.chain.getFinalizedCheckpoint();
     const slot = signedBlock.message.slot;
     // block is too old
     if (slot <= computeStartSlotAtEpoch(this.config, finalizedCheckpoint.epoch)) {
-      return false;
+      return ExtendedValidatorResult.ignore;
     }
 
     const parentBlock = await this.db.block.get(signedBlock.message.parentRoot.valueOf() as Uint8Array);
-    if(!parentBlock) {
-      return false;
-    }
-    const {state, epochCtx} = await this.db.stateCache.get(parentBlock.message.stateRoot);
```

Comment:
need confirmation

Reply:
I think we start doing validations against the head state, then, second-to-last rule (and last rule) is the check against the parent block.

Reply:
ok so I use the head state here.

I remove the check of `parentBlock` because:
+ it is not part of the spec
+ it causes the issue of not being able to sync to head after initial sync